components/NavigationBar/src/MDCNavigationBar.m

```diff
@@ -448,7 +448,11 @@ - (CGRect)mdc_frameAlignedVertically:(CGRect)frame
     }
 
     case UIControlContentVerticalAlignmentTop: {
-      return CGRectMake(frame.origin.x, bounds.origin.y, frame.size.width, frame.size.height);
+      CGFloat navigationBarCenteredY =
+          MDCFloor(([self intrinsicContentSize].height - frame.size.height) / 2);
+      navigationBarCenteredY = MAX(0, navigationBarCenteredY);
```

Comment:
This seems surprising.  Why am I using centerY is I'm AlignmentTop.  Can you add a comment to clarify?  Something along the lines:
The title frame is vertically centered with the back button but will stick to the top of the header regardless of the header's height.